pkgs/by-name/po/postfixadmin/package.nix

```diff
@@ -1,30 +1,48 @@
-{ fetchFromGitHub, stdenv, lib, nixosTests }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch2,
+  nixosTests,
 
-stdenv.mkDerivation rec {
+  configLocalPath ? "/etc/postfixadmin/config.local.php",
+  templatesCachePath ? "/var/cache/postfixadmin/templates_c",
+}:
+stdenv.mkDerivation (finalAttrs: {
   pname = "postfixadmin";
-  version = "3.3.13";
+  version = "3.3.14";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "${pname}-${version}";
-    sha256 = "sha256-46bc34goAcRvaiyW7z0AvIcd8n61TL6vgLQ+y7nNKBQ=";
+    owner = "postfixadmin";
+    repo = "postfixadmin";
+    rev = "refs/tags/postfixadmin-${finalAttrs.version}";
+    sha256 = "sha256-T7KRD0ihtWcvJB6pZxXThFHerL5AGd8+mCg8UIXPZ4g=";
   };
 
+  patches = [
+    # Fix https://github.com/postfixadmin/postfixadmin/issues/872
+    (fetchpatch2 {
+      url = "https://github.com/postfixadmin/postfixadmin/commit/8ec9140673afd9996a3f81cca600ea0e5bd31cf8.patch";
```

Comment:
@DavidGoodwin is it ok to apply this commit as a patch to 3.3.14?

Reply:
@Kamillaova yes.